yaml/_mark.ts

```diff
@@ -20,59 +24,42 @@ export class Mark {
     this.column = column;
   }
 
-  getSnippet(indent = 4, maxLength = 75): string | null {
-    if (!this.buffer) return null;
```

Comment:
Is it safe to remove this check?

Reply:
Realistically, this is safe to remove, unless the user disables or misuses TypeScript. Either way, I've reverted this for now.